frontend/src/container/ExplorerOptions/ExplorerOptions.tsx

```diff
@@ -80,11 +134,13 @@ function ExplorerOptions({
 	};
 
 	const onUpdateQueryHandler = (): void => {
+		const extraData = viewsData?.data.data.find((view) => view.uuid === viewKey)
+			?.extraData;
 		updateViewAsync(
 			{
 				compositeQuery: mapCompositeQueryFromQuery(currentQuery, panelType),
 				viewKey,
-				extraData: '',
+				extraData: extraData || JSON.stringify({ color: Color.BG_SIENNA_500 }),
```

Comment:
is this expected? extra data is stringify color?

Reply:
Yup, we don't have any specific property for setting up the colours.